charts/render.py

```diff
@@ -72,7 +72,7 @@ def maybe_copy(doc, src, dest):
 
 
 def fixup_dict(doc):
-    if doc.get('heritage') == 'Tiller' or \
+    if doc.get('heritage') in ('Helm', 'Tiller') or \
             doc.get('app.kubernetes.io/managed-by') == 'Tiller':
```

Comment:
I think you want to change this one as well

Reply:
done